### PR TITLE
[translations] add additional Polish translations and fix pluralizations

### PR DESCRIPTION
**Pull request by Flower7C3**

### CODE REVIEW

File: src/Resources/translations/ResetPasswordBundle.pl.xlf [hunk 1]
```diff
@@ -1,26 +1,50 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" version="1.2">
-    <file original="file.ext" source-language="en" target-language="pl" datatype="plaintext">
+    <file original="file.ext" source-language="pl" target-language="pl" datatype="plaintext">
```
Comment (bocharsky-bw) [inline src/Resources/translations/ResetPasswordBundle.pl.xlf:3]: This line should be reverted, the source language is `en` indeed

**Review by bocharsky-bw — CHANGES_REQUESTED** [review]
Thank you for fixing bad pluralization here and adding missing translations! The only comment I noticed that should be fixed before we could merge this

**Comment by bocharsky-bw** [comment]
@Flower7C3 P.S. If you have time, please, take a look at another bundle that also misses some Polish translations: https://github.com/SymfonyCasts/verify-email-bundle/blob/main/src/Resources/translations/VerifyEmailBundle.pl.xlf 

It would be great to add missing translations there too :)

**Comment by Flower7C3** [comment]
Source language fixed.

**Review by bocharsky-bw — APPROVED** [review]
Dziękuje bardzo!

**Review by jrushlow — APPROVED** [review]
Thanks for the updates @Flower7C3 !

**Pull request merged**